docs/001_develop/03_client-capabilities/018_state-management/index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: 'State management'
+sidebar_label: 'State management'
+id: client-state-management
+keywords: [genesis, foundation, redux, state management, redux state management, store, actions, reducers]
+tags:
+  - redux
+  - state management
+  - redux state management
+  - redux store
+  - redux store management
+  - actions
+  - reducers
+  - middleware
+  - devtools
+  - communication
+  - redux
+  - injection
+  - state
+  - reducers
+  - middleware
+  - devtools
+sidebar_position: 18
+---
+
+# State management
+
+Manage application state in Genesis apps with either our built-in store or a Redux Toolkit–powered alternative. Pick what suits your team and scale, then dive in.
+
+## [Foundation Store](./01_foundation-store)
+
+The Foundation Store is a lightweight, DI-friendly store designed for Genesis components. It uses strongly typed CustomEvents, reducers via `createListener`, and derived data via getters. Split state into store fragments for clean domain boundaries and testability.
+
+Start here if you want tight Genesis integration with minimal external dependencies.
+
+## [Foundation Redux](./02_foundation-redux)
```

Comment:
Jay has been saying we need to use absolute rather than relative links. Update that and any other places where they're used (they're ok in the API docs)